Dockerfile

```
@@ -0,0 +1,33 @@
+# Файл для тестов с запуском без docker-compose
+
+ARG NODE_VERSION=20
+ARG SERVER_PORT=3000
+
+FROM node:$NODE_VERSION-buster AS base
+
+WORKDIR /app
+
+FROM base AS builder
+
+COPY package.json yarn.lock ./
+RUN yarn install --frozen-lockfile
+# RUN yarn install
+
+COPY . .
+
+RUN yarn lerna bootstrap
+RUN rm -rf /app/packages/server/dist/ && yarn build --scope=server
```

```suggestion
RUN rm -rf /app/packages/server/dist/ && yarn build --scope=server
```

Можно удалить очистку каталога, она заложена в команду build сервера